packages/drawer/src/DrawerBase.tsx

```diff
@@ -0,0 +1,84 @@
+import Button from '@littlespoon/button'
+import { up, xs } from '@littlespoon/theme/lib/breakpoints'
+import { shadeWhite } from '@littlespoon/theme/lib/colors/token'
+import { family } from '@littlespoon/theme/lib/fonts/primary'
+import { rem } from '@littlespoon/theme/lib/utils'
+import styled from 'styled-components'
+
+import { DrawerProps } from './Drawer'
+
+export const DrawerBase = styled.div<Partial<DrawerProps>>`
+  align-content: stretch;
+  background-color: ${shadeWhite};
+  border: 0;
+  border-radius: ${rem(1.2)} ${rem(1.2)} 0 0;
+  bottom: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+  left: 0;
+  margin: 0;
+  max-height: 80%;
+  max-width: none;
+  /* TODO: Update this value */
+  min-height: ${rem(20)};
+  min-width: unset;
+  outline: none;
+  overflow: hidden;
+  padding: ${rem(0.8)} ${rem(2)} ${rem(2)};
+  position: fixed;
+  right: 0;
+  /* TODO: Update z-index value? */
```

Comment:
Resolve todo?

Reply:
@remarkablemark Please read https://github.com/little-spoon-dev/design-system/pull/604/#discussion_r959729558

Reply:
Implemented in https://github.com/little-spoon-dev/design-system/pull/609/commits/31ace5627eade9ea720fd18257e09cdcdee39dc0